js/src/Map.js

```diff
@@ -54,7 +69,8 @@ var LeafletMapModel = widgets.DOMWidgetModel.extend({
         options : [],
         layers : [],
         controls : [],
-        crs: 'EPSG3857'
+        crs: 'EPSG3857',
+        style: 'move'
```

Comment:
Here style must be an instance of `LeafletMapStyleModel`, maybe put `style: null` for now.